test/extended/authentication.sh

```diff
@@ -59,34 +59,135 @@ oc new-project openldap
 # create all the resources we need
 oc create -f test/extended/fixtures/ldap
 
+is_event_template=(                      \
```

Comment:
ow, my eyes... no tabs, single quote to avoid escaping $ and ", and line up the trailing \'s?
```
server_ready_template=(                                \
'{{with $items := .items}}'                            \
  '{{with $item := index $items 0}}'                   \
    '{{with $map := index $item.status.conditions 0}}' \
      '{{with $state := index $map "type"}}'           \
        '{{$state}}'                                   \
      '{{end}}'                                        \
      '{{with $valid := index $map "status"}}'         \
        '{{$valid}}'                                   \
      '{{end}}'                                        \
    '{{end}}'                                          \
  '{{end}}'                                            \
'{{end}}'                                              \
)
```